.github/workflows/release-apps.yml

```
@@ -0,0 +1,52 @@
+name: Release Apps
+
+on:
+  release:
+    types: [ published ]
+  push:
+    branches: [ main ]
+  pull_request_target:
+    types: [ opened, edited, synchronize, reopened ]
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Release Apps
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        app: [ legacy-api, ocean-api ]
+    steps:
+      - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@230611dbd0eb52da1e1f4f7bc8bb0c3a339fc8b7
```
better
https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions